db/migrate/20200526152251_adds_fields_to_people.rb

```diff
@@ -0,0 +1,18 @@
+class AddsFieldsToPeople < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
maybe add indexes on:
* prison_number
* criminal_records_office
* police_national_computer
* last_name
* date_of_birth
?

Reply:
So we're filtering on the identifiers so I don't think we need it for the rest. We should add indexes when we need them, I think

Reply:
👍